Extensions/quick_tags.js

```diff
@@ -1,7 +1,7 @@
 //* TITLE Quick Tags **//
-//* VERSION 0.5.6 **//
-//* DESCRIPTION Quickly add tags to posts **//
-//* DETAILS Allows you to create tag bundles and add tags to posts without leaving the dashboard. **//
+//* VERSION 0.6.0 **//
+//* DESCRIPTION Quickly edit the tags of posts **//
+//* DETAILS Allows you to create tag bundles and edit the tags of posts without leaving the dashboard. **//
 //* DEVELOPER STUDIOXENIX **//
```

Comment:
This isn't a serious problem, but while you're revising, if you could change STUDIOXENIX to new-xkit, that'd be great.